help.txt

```diff
@@ -219,20 +231,66 @@ FUNCTIONS
         - `smart_open README.rst
           <https://github.com/RaRe-Technologies/smart_open/blob/master/README.rst>`__
     
+    parse_uri(uri_as_string)
+        Parse the given URI from a string.
+        
+        Parameters
+        ----------
+        uri_as_string: str
+            The URI to parse.
+        
+        Returns
+        -------
+        collections.namedtuple
+            The parsed URI.
+        
+        Notes
+        -----
+        Supported URI schemes are:
+        
+        * file
```

Comment:
```suggestion
        * file (default, used when no scheme specified)
```

Reply:
I gradually came up with this from scratch myself, but I'm sure it has been done before (e.g. self-documenting plugins).